RedHat-UX/red-hat-design-system
# bug(cta): add aria-hidden to arrow icon (#729)

## What I did

1. Added an `aria-hidden="true"` attribute to the default CTA arrow `<svg>` element, in order to hide the icon from assistive tech. (The change is in line 106 of rh-cta.ts.)


## Testing Instructions

1. [Visit the demo page](https://deploy-preview-732--red-hat-design-system.netlify.app/components/call-to-action/demo/) with a browser in which this bug occurred (e.g., Mac Chrome or Firefox).
2. Open a screen reader. (On Mac, open VoiceOver with Command-F5 or by holding Command plus three rapid presses on Touch ID.)
3. Tab until focus is on the "Default" call-to-action link:
![Default link with focus](https://user-images.githubusercontent.com/642695/218861984-20a4fc06-2d4b-49f8-8281-565b0d54e62e.png)
4. Instruct the screen reader to start reading the page (Control-Option-A with Mac VoiceOver).
6. As your screen reader reads down the page, note whether it says the word "image" after reading the words "link, Default" for the default call-to-action links. To pass this test, it should **not** say "image," as the arrow is now intended to be ignored by assistive tech. (Pressing the Control key stops Mac VoiceOver from reading at any point.)
7. You may also right-click and inspect the arrow `<svg>`, to ensure it now includes the `aria-hidden="true"` attribute.

## Notes to Reviewers
Thanks!
